TestCases/rans/naca0012/turb_NACA0012_sst_fixedvalues.cfg

```diff
@@ -0,0 +1,260 @@
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%                                                                              %
+% SU2 configuration file                                                       %
+% Case description: upstream turbulence quantities fixed to far-field value    %
+% Date: Mar 17th, 2021                                                         %
+% File Version 7.1.1 "Blackbird"                                               %
+%                                                                              %
```

Comment:
I think it is a nice touch to put your name (or some identifier) here so it is directly clear who did this :) If it is intentional for privacy reasons then sure, but as you seem to use your actual name in github i guess thats not the issue here

Reply:
I modified the `turb_NACA0012_sst_sust.cfg` file by @economon, so I was unsure which author name(s) and institution(s) to use. Would it be appropriate to write

    % Author: Max Aehle
    % Institution: TU Kaiserslautern